google/cloud/common_options.h

```diff
@@ -91,6 +91,40 @@ struct UserProjectOption {
   using Type = std::string;
 };
 
+/**
+ * Configure the QuotaUser [system parameter].
+ *
+ * A pseudo user identifier for charging per-user quotas. If not specified, the
+ * authenticated principal is used. If there is no authenticated principal, the
+ * client IP address will be used. When specified, a valid API key with service
+ * restrictions must be used to identify the quota project. Otherwise, this
+ * parameter is ignored.
+ *
+ * [system parameter]: https://cloud.google.com/apis/docs/system-parameters
+ *
+ * @ingroup options
+ * @ingroup rest-options
+ */
+struct QuotaUserOption {
+  using Type = std::string;
+};
+
+/**
+ * Configure the UserIp system parameter.
```

Comment:
Was "system parameter" supposed to be a link?

Reply:
Yes, fixed in #13937